sunrisemvmtnyc/legislation-tracker
# chore: "Advanced page" design improvements

### To test:
Visit "advanced page". Verify you can see table. Should be scrollable and filterable by bills & legislators. Should show legislator bill percentage on left.

Some MVP needed improvements to make the page usable:

- [x] column header - use printno & show title on hover
- [x] horizontal axis text box filter bill
- [x] vert axis text box filter legislator
- [x] css improvements: clearer divisions
- [x] show column header even while scrolled
- [x] show leg name even while scrolled
- [x] campaign super-header
- [x] all leg pct & climate leg pct
